security/login_link.rst

```diff
@@ -0,0 +1,510 @@
+.. index::
+   single: Security; Login link
+   single: Security; Magic link login
+
+How to use Passwordless Login Link Authentication
+=================================================
+
+Login links, also called "magic links", allow users to login without
+passwords. Whenever a user wants to login, a login link is generated and
+send to the user (e.g. using an e-mail). The user is logged in as soon as
+they visit that link.
+
+Login links are supported by Symfony when using the experimental
+authenticator system. You must
+:ref:`enable the authenticator system <security-enable-authenticator-manager>`
+in your configuration to use this feature.
+
+Using the Login Link Authenticator
+----------------------------------
+
+This guide assumes you have set-up security and have created a user object
```

Comment:
Not sure, but I think: `set-up` -> `set up` because here it's a verb instead of a noun.